src/main/java/uk/ac/ebi/subs/fileupload/listeners/ValidationMessageByFileDeletion.java

```diff
@@ -0,0 +1,9 @@
+package uk.ac.ebi.subs.fileupload.listeners;
+
+import lombok.Data;
+
+@Data
+public class ValidationMessageByFileDeletion {
```

Comment:
I'm not sure what this class name means

Reply:
FileReferenceValidationMessage would be better?